.drone.yml

```diff
@@ -0,0 +1,22 @@
+kind: pipeline
+name: default
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: test
+  image: golang
+  privileged: true
+  environment:
+    COVERALLS_TOKEN:
+      from_secret: coveralls_token
+  commands:
+  - apt-get -q -y update
+  - apt-get -q -y install lsof net-tools iptables
+  - iptables -D ufw-before-input -m conntrack --ctstate INVALID -j DROP || echo "Rule not found"
```

Comment:
This will always exit 0, so will this cause Drone to miss a failure here?

Reply:
It's intentional. We just want to make sure the rule isn't present before we run, whether or not it was previously present.

Reply:
Ah, I see.  I didn't realize the -D flag was to delete a rule.  As I said in another comment, I'd like to learn more about iptables haha